cpp/include/cudf/aggregation.hpp

```diff
@@ -64,5 +64,24 @@ std::unique_ptr<aggregation> make_median_aggregation();
  */
 std::unique_ptr<aggregation> make_quantile_aggregation(
     std::vector<double> const& q, interpolation i);
+
+/**
+ * @brief Factory to create an `argmax` aggregation
+ * 
+ * `argmax` returns the index of the maximum element.
```

Comment:
Do you mean return type of the factory or the return type of `argmax`? For factory, the return type is clear from the doxy brief. For the return type of `argmax`, as is the case with other aggregations, the return type is decided by `target_type_t` in `aggregations.hpp` and not mentioned here because it depends on Source type.

Reply:
@rgsl888prabhu I believe you are conflating what the factory returns with what an `argmax` aggregation does. This documentation describes the latter.